src/consumerIncentives/ConsumerIncentivesHomeScreen.tsx

```diff
@@ -157,24 +166,33 @@ function SuperchargingInfo() {
   )
 }
 
-function ClaimSuperchargeRewards({ rewards }: { rewards: SuperchargePendingReward[] }) {
+function ClaimSuperchargeRewards({
+  rewards,
+}: {
+  rewards: SuperchargePendingReward[] | SuperchargePendingRewardV2[]
+}) {
   const tokens = useSelector(tokensByAddressSelector)
   const { t } = useTranslation()
 
-  const rewardsByToken: { [tokenAddress: string]: BigNumber | undefined } = rewards.reduce(
-    (acc, reward) => {
+  const rewardsByToken: { [tokenAddress: string]: BigNumber } = {}
+
+  if (isSuperchargePendingRewardsV2(rewards)) {
+    rewards.forEach((reward) => {
+      const tokenAddress = reward.details.tokenAddress.toLowerCase()
+
+      rewardsByToken[tokenAddress] = (rewardsByToken[tokenAddress] || new BigNumber(0)).plus(
+        new BigNumber(reward.details.amount).div(WEI_PER_TOKEN)
```

Comment:
nit: This could be incorrect if we decided to reward with tokens not using 18 decimals.
I know this just copies what used to be done. Just noting it.